src/chunk_scan.c

```diff
@@ -179,10 +186,17 @@ ts_chunk_scan_by_constraints(const Hyperspace *hs, const List *dimension_vecs,
 		MemoryContextSwitchTo(orig_mcxt);
 		MemoryContextDelete(work_mcxt);
 
+		Chunk *osm_chunk = NULL;
+		if (osm_chunk_id != INVALID_CHUNK_ID)
+		{
+			osm_chunk = ts_chunk_get_by_id(osm_chunk_id, true);
+			chunks = MemoryContextAlloc(orig_mcxt, sizeof(Chunk *) * 1);
```

Comment:
```suggestion
			chunks = MemoryContextAlloc(orig_mcxt, sizeof(Chunk *));
```
Do we need to multiply by 1?